src/islet/islet_shim.cc

```diff
@@ -69,33 +69,33 @@ bool cca_Verify(const int what_to_say_size, byte* what_to_say,
     return false;
 
   int len = digest_output_byte_size("sha-256");
-  byte cca_what_to_say_expected[len];
+  byte islet_what_to_say_expected[len];
   if (!digest_message("sha-256", what_to_say, what_to_say_size,
-        cca_what_to_say_expected, len)) {
-    printf("cca_Verify: Can't digest what_to_say\n");
+        islet_what_to_say_expected, len)) {
+    printf("islet_Verify: Can't digest what_to_say\n");
     return false;
   }
 
-  byte cca_what_to_say_returned[2*len];
+  byte islet_what_to_say_returned[2*len];
   int user_data_len = len;
-  rv = islet_parse(CLAIM_TITLE_USER_DATA, claims, claims_len, cca_what_to_say_returned, &user_data_len);
+  rv = islet_parse(CLAIM_TITLE_USER_DATA, claims, claims_len, islet_what_to_say_returned, &user_data_len);
   if (rv != ISLET_SUCCESS)
     return false;
 
-  if (memcmp(cca_what_to_say_returned, cca_what_to_say_expected, len) != 0)
+  if (memcmp(islet_what_to_say_returned, islet_what_to_say_expected, len) != 0)
     return false;
 
   rv = islet_parse(CLAIM_TITLE_RIM, claims, claims_len, measurement_out,
                    measurement_out_size);
   return rv == ISLET_SUCCESS;
 }
 
-bool cca_Seal(int in_size, byte* in, int* size_out, byte* out) {
+bool islet_Seal(int in_size, byte* in, int* size_out, byte* out) {
```

Comment:
Same 'issue': `islet_Seal()` is too close to lower level API `islet_seal()`.

Reply:
One option for you to consider for the future is maybe to name the lower-level APIs as one of:

- `islet_seal_()`
- `_islet_seal()`
- `islet__seal()`

None of these are really attractive. I'll let @bitboom and @jinbum think about this separately.